src/pages/ProductListPage/ProductListPage.jsx

```diff
@@ -49,218 +65,16 @@ export const ProductListPage = () => {
                 author={item.author} 
                 price={item.price}
                 discount={item.discount}  
-                discountedPrice={item.discountedPrice} />
+                discountedPrice={item.discountedPrice}
+                rating={item.rating}
+                productType={item.productType}
+                feature={item.feature}
+                stockType={item.stockType}
+                stock={item.stock}
+                categoryType={item.categoryType} />
```

Comment:
pass all these props in a single object say : item and then destructure it in ProductCards, becuase anyway you will have to destructure it in ProductCard.

Reply:
ok